src/components/radioButton/RadioGroup.tsx

```diff
@@ -0,0 +1,65 @@
+import _ from 'lodash';
```

Comment:
For next migration, make sure to not delete and create new file but to make sure git just rename the files 
(similar to how u did for `RadioButton` file)
It's important for git log history of the files

Reply:
I reverted and renamed, but I think it just looks different in GitHub, I see the history locally, can you verify you do to?